cfme/services/catalogs/catalog_item.py

```diff
@@ -223,9 +223,10 @@ def create(self):
             # Address BZ1321631
             tabstrip.select_tab("Environment")
             tabstrip.select_tab("Catalog")
+            prov_row_title = 'Provider' if version.current_version() < "5.7" else 'Deprecated'
```

Comment:
This is a column bug in ManageIQ, so we _can_ merge this, but we should document it BIG TIME! I also just merged a PR from @sshveta which added a blocker for this particular issue to the test so it doens't get run, if we are going to put a hack in for this, we should probably remove the blocker in the same commit.